lua/roslyn/lsp.lua

```diff
@@ -89,6 +89,44 @@ function M.start(bufnr, root_dir, on_init)
         end
     end
 
+    vim.api.nvim_create_autocmd({ "BufReadCmd" }, {
+        pattern = "roslyn-source-generated://*",
+        callback = function()
+            local uri = vim.fn.expand('<amatch>')
+            local buf = vim.api.nvim_get_current_buf()
+            vim.bo[buf].modifiable = true
```

Comment:
If I understand it correctly, we only set modified here to be able to set the lines in the handler, and then immediately set it to nomodifiable again after that?

Is it possible to set it to modifiable just before setting the lines inside the handler or doesn't that work?

Reply:
I see that this is also done for the jdtls, so it is probably fine🤷‍♂️ I was just curious